### PR TITLE
Don't add more-info-card if hidden

### LINKED ISSUE #460: aesthetic: state-card also has cursor:pointer if domain is listed in HIDE_MORE_INFO
https://github.com/home-assistant/home-assistant-polymer/blob/19187ce518190823a8ccc5e7fc3d262cd218fa74/src/cards/ha-entities-card.html#L18

https://github.com/home-assistant/home-assistant-polymer/blob/master/src/util/hass-util.html#L34

### LINKED ISSUE #463: glitch: state-card: open more-info even if domain is in HIDE_MORE_INFO
if you click on the marked area you can still open more-info-card. even if the domain is listed in HIDE_MORE_INFO 
state-card2

### PR DESCRIPTION
**Pull request by c727**
replaces #472

fix #460
fix #463

### CODE REVIEW

File: src/cards/ha-entities-card.html [hunk 1]
```diff
@@ -90,8 +92,25 @@
...
+
+  addTapEvents() {
+    var cards = this.root.querySelectorAll('.has-more-card');
+    for (var i of cards) {
+      i.addEventListener('tap', this.entityTapped);
+    }
+  },
 
   entityTapped(ev) {
+    alert(ev.target.nodeName + "\n" + ev.model);
```
Comment (c727) [inline src/cards/ha-entities-card.html]: ev.model is undefined, how can I solve this?

File: src/cards/ha-entities-card.html [hunk 2]
```diff
@@ -91,7 +93,19 @@
...
+      i.addEventListener('tap', this.entityTapped);
+    }
+  },
+
+  computeStateClass(stateObj) {
+    return (window.hassUtil.stateMoreInfoType(stateObj) !== 'hidden' ? 'state has-more-card' : 'state');
+  },
+
   entityTapped(ev) {
+    alert(ev.target.nodeName + "\n" + ev.model);
```
Comment (c727) [inline src/cards/ha-entities-card.html]: ev.model is undefined, how can I solve this?

Comment (c727) [reply src/cards/ha-entities-card.html]: ...Or we use [this (working)](https://github.com/home-assistant/home-assistant-polymer/compare/master...c727:patch-5). Any feedback would be great

**Comment by andrey-git** [comment]
https://www.polymer-project.org/2.0/docs/devguide/templates#handling-events

You can use modelForElement to get the model.
In this case you only need the model to get the item, so use `itemForElement`

**Comment by c727** [comment]
thanks Andrey :-)

**Comment by c727** [comment]
@andrey-git @balloob some feedback would be nice.
If you don't like all the added code, I can limit it to 2x dom-if instead

**Review by andrey-git — APPROVED** [review]
Looks good, but lets wait for @balloob for merging

**Pull request merged**